ca_driver/launch/create_1.launch

```diff
@@ -8,5 +8,6 @@
     <param name="~create_1" value="true" />
   </node>
 
-  <!-- TODO: Create 1 robot description -->
+  <!-- Robot description -->
+  <include if="$(arg desc)" file="$(find ca_description)/launch/create_1.launch"/>
```

Comment:
You should add the line `<arg name="desc" default="true" />` near the top of this file.